src/components/Header/UserMenu.jsx

```diff
@@ -57,7 +62,7 @@ export const UserMenu = () => {
           borderColor: textColor
         }}
       >
-        <SettingsIcon w="25px" h="25px" />
+        <Avatar name={userName} w="40px" h="40px" />
```

Comment:
As we use data from the backend, it's better to add <Skeleton> or <Preloader> while the data are not fetched

Reply:
yes agree added Skeleton circle